csatar-plugins/csatar/models/Scout.php

```diff
@@ -24,4 +24,28 @@ class Scout extends Model
      */
     public $rules = [
     ];
+
+    /**
+     * Relations
+     */
+    public $hasOne = [
+        'legal_relationship' => '\Csatar\Csatar\Models\LegalRelationship',
+        'special_diet' => '\Csatar\Csatar\Models\SpecialDiet',
+        'religion' => '\Csatar\Csatar\Models\Religion',
+        'tshirt_size' => '\Csatar\Csatar\Models\TShirtSize'
+    ];
+
+    public $belongsToMany = [
+        'chronic_illnesses' => [
+            '\Csatar\Csatar\Models\ChronicIllness',
+            'table' => 'csatar_csatar_scouts_chronic_illnesses',
+            'order' => 'title'
```

Comment:
Why do you add `'order' => 'title'` here and on other `belongsToMany` relations?